### PR TITLE
Sorting enabled for 'receipt_date', 'committee_id', and 'file_number'.

### PR DESCRIPTION
**Pull request by jontours**
Basically commented out sorting argument code and added correct columns. Is ready to go.

### CODE REVIEW

**Comment by noahmanger** [comment]
Does something need to run before this works? Trying it out locally and it's not working.

File: webservices/args.py [hunk 1]
```diff
@@ -223,6 +223,8 @@ def make_seek_args(field=fields.Int, description=None):
 efilings = {
     'file_number': fields.List(fields.Int, description=docs.FILE_NUMBER),
     'committee_id': fields.List(IStr, description=docs.COMMITTEE_ID),
+    'report_id': fields.Int(),
+    'receipt_date': fields.Date(),
```
Comment (LindsayYoung) [inline webservices/args.py]: we already have `min_reciept_date` and `max_reciept_date` so I don't think we want `receipt_date` too

**Comment by jontours** [comment]
@noahmanger there will be errors on sorting for this until we add the indices.

File: webservices/args.py (hunk 1 above)
Comment (jontours) [inline webservices/args.py]: Yes you are right!  I forgot that was is passed as the sort parameter determines the sorting column, with that being said the rest of the parameters can be pulled.

**Pull request merged**